jOOL-java-8/src/main/java/org/jooq/lambda/Agg.java

```diff
@@ -979,4 +979,694 @@ else if (percentile == 1.0)
             s -> s.map(Objects::toString).orElse("")
         );
     }
+
+    //CS304 Issue link: https://github.com/jOOQ/jOOL/issues/151
+
+    /**
+     * Calculate the linear regression function Slope of the given object collectors,
+     * based on the mapping function from object to double number.
+     *
+     * <p><pre> Usage of aggregation function regrSlopeBy():
+     * The mapping functions functionX and functionY are functions mapping the objects to double values X and Y, for instance:
+     * {@code
+     * FunctionX<Tuple2<Integer,Integer>, Double> mapping = e -> Double.valueOf(e.v1);
+     * FunctionY<Tuple2<Integer,Integer>, Double> mapping = e -> Double.valueOf(e.v2);
+     * }
+     * The specific usage of varianceBy is like:
+     * {@code
+     * Seq.of(new Tuple2<Integer,Integer>(1,1),new Tuple2<Integer,Integer>(2,2)).collect(Agg.regrSlopeBy(functionX,functionY));
+     * }
+     * Besides, self defined class is also allowed with mapping function:
+     * {@code
+     * Seq.of(new Node(-1,1),new Node(1,1)).collect(Agg.regrSlopeBy(functionX,functionY));
```

Comment:
The `Node` class from the tests shouldn't be mentioned here, it is confusing readers. We have `Tuple2` types that are better suited (I think?)

Reply:
Same, let's just remove all Javadoc for the time being.